Assets/Scripts/LightStatus.cs

```diff
@@ -2,14 +2,67 @@
 using System.Collections.Generic;
 using UnityEngine;
 
+public enum LightType { DEFAULT, RED, BLUE };
+
 public class LightStatus : MonoBehaviour
 {
+    [Header("Common")]
+    [SerializeField]
+    private int collisionDamage;
+    [SerializeField]
+    private LightType lightType;
+
+    [Header("RedLight")]
+    [SerializeField]
+    private int burningDamage;
+    [SerializeField]
+    private float damageTime;
+    [SerializeField]
+    private float burningTime;
+
+    [Header("BlueLight")]
     [SerializeField]
-    private float LightDamage = 1;
-    public float Light_currentDamage;
+    public float slowRate;
 
-    private void Awake()
+    private GameObject Monster;
+
+    private void OnTriggerEnter2D(Collider2D collision)
+    {
+        if (collision.CompareTag("Monster"))
+        {
+            Monster = collision.gameObject;
```

Comment:
Monster가 메서드 바깥에서 공유하지 않는 것 같은데 내부에서만 사용하게 변경하는 건 어떨까요?
```suggestion
            GameObject Monster = collision.gameObject;
```

Reply:
넵 위의 내용들 말씀해주신대로 변경하도록 하겠습니다!